engine/getstarted/linux_install_help.md

```diff
@@ -3,36 +3,65 @@ description: Getting started with Docker
 keywords: beginner, getting started, Docker, install
 redirect_from:
 - /mac/started/
-title: 'Example: Install Docker on Ubuntu Linux'
+title: Install Docker using a shell script
 ---
 
-This installation procedure for users who are unfamiliar with package
-managers, and just want to try out the Getting Started tutorial while running Docker on Linux. If you are comfortable with package managers, prefer not to use
-`curl`, or have problems installing and want to troubleshoot, please use our
-`apt` and `yum` <a href="/engine/installation/"
-target="_blank">repositories instead for your installation</a>.
+This installation procedure for users who don't want to use a package manager to
+install Docker. If you can use a package manager, you should use the
+[recommended installation procedure for your operating system](/engine/installation/)
+instead. Using a package manager ensures that you get upgrades when they are
+available, and allows you to install a specific version of Docker, rather than
+the very latest version.
```

Comment:
Probably for a follow-up, but we should point out that the script is working on a "best effort" base, both to determine the operating system, and to configure docker with a default configuration "that works", but may not be tuned for production.